recipes/scons/all/test_package/.gitignore

```diff
@@ -0,0 +1 @@
+*.dblite
```

Comment:
Remove the gitignore file

Reply:
scons creates a `.sconsign.dblite`  file when building.
The `.gitignore` is there to avoid `git` picking it up.

Reply:
so change the root gitignore in CCI repo

Reply:
@madebr in SCons manual, sconsign.dblite can be changed, so you can move it to test_package/<hash>/build folder: https://scons.org/doc/3.1.2/HTML/scons-man.html

Reply:
In the mean time, `*.dblite` has been added to  to the global `.gitignore` in ba2b7bf8e.
But I fixed the location of this signature file thanks to your tip.